toolkit/tools/imagegen/installutils/installutils.go

```diff
@@ -641,6 +641,7 @@ func TdnfInstallWithProgress(packageName, installRoot string, currentPackagesIns
 		const tdnfInstallPrefix = "Installing/Updating: "
 
 		// Only process lines that match tdnfInstallPrefix
+		logger.Log.Debug(line)
```

Comment:
If it is desirable to put all the stdout lines in Debug, then this code can be greatly simplified. Specifically, `LogLevel(logrus.TraceLevel, logrus.WarnLevel)` below can be changed to `LogLevel(logrus.DebugLevel, logrus.WarnLevel)`. And then most of the `onStdout` function can be deleted.

But that was intentionally not done because the tdnf's stdout can be very spammy.